src/interfaces/controllers/insert-trailer-registration/insert-trailer-validator.ts

```diff
@@ -0,0 +1,9 @@
+import Joi from 'joi';
+
```

Comment:
I think it probably make more sense if move that in a validator folder under interfaces

Reply:
moved